CONTRIBUTING.md

```diff
@@ -37,13 +37,21 @@ If you are modifying a file in the `client/` directory, be sure to run `npm run
 
 Pull requests often need some real-world testing.
 
-1. In your `package.json`, change the line with `webpack-dev-server` to:
+1a. In your `package.json`, change the line with `webpack-dev-server` to:
 
   ```json
   "webpack-dev-server": "github:webpack/webpack-dev-server#pull/<ID>/head"
   ```
 
   `<ID>` is the ID of the pull request.
+OR
```

Comment:
Can we add docs for [`npm/yarn link`](https://docs.npmjs.com/cli/link) usage here instead?  You can keep `  "webpack-dev-server": "file:../path/to/local/webapck-dev-server/fork"` if you like, but please incoporate it into the first example as an alternative **value**

Reply:
I could update the it with `link` but I'd prefer to do it with another PR, if it's ok